scout/index.md

```diff
@@ -16,62 +16,60 @@ redirect_from:
 
 {% include scout-early-access.md %}
 
-Docker Scout is a collection of software supply chain features that appear
-throughout Docker user interfaces and the command line interface (CLI). These features provide detailed
-insights into the composition and security of container images.
+Container images are often built from layers of other container images and
+software packages. These layers and packages can contain vulnerabilities that
+make your containers and the applications they run vulnerable to attack.
 
-Docker Scout analyzes image contents and generates a detailed report of
-packages and vulnerabilities that it detects. Docker Scout can also help provide
-you with suggestions for how you can remediate issues discovered by the image
-analysis.
+Docker Scout can proactively help you find and fix these vulnerabilities,
+helping you create a more secure software supply chain.
 
-The [image details view](./image-details-view.md) in Docker Desktop and the tag
-details pages on Docker Hub are both powered by Docker Scout.
+It does this by analyzing your images and creating a full inventory of the
```

Comment:
Suggest moving this section to the paragraph above or clarify what we mean by "It" in the "It does this.." and "It then correlates".